msteams-platform/resources/schema/manifest-schema.md

```diff
@@ -936,7 +936,7 @@ Delegated permissions allow the app to access data on behalf of the signed-in us
     |`ChannelMeetingAudioVideo.Stream.Group`| Allows the app to stream audio-video content from channel meetings associated with the team. |
     |`InAppPurchase.Allow.Group`| Allows the app to show marketplace offers to users in the team and complete their purchases within the app, on behalf of the signed-in user.|
     |`ChannelMeetingStage.Write.Group`| Allows the app to show content on the meeting stage in channel meetings associated with the team, on behalf of the signed-in user.|
-    |`LiveShareSession.ReadWrite.Group`|Allows the app to create and synchronize Live Share sessions for meetings associated with the team, and access related information about the meeting's roster, such as member's meeting role, on behalf of the signed-in user.|
+    |`LiveShareSession.ReadWrite.Group`|Allows the app to create and synchronize Live Share sessions for the team. Provide access related information about the meeting's roster, such as member's meeting role, on behalf of the signed-in user.|
```

Comment:
Allows the app to create and synchronize Live Share sessions for the team as well as, for any meetings associated with the team, provide access related information about the meeting's roster, such as member's meeting role, on behalf of the signed-in user.

Reply:
Updated the doc as suggested